src/abstractdataframe/show.jl

```diff
@@ -113,7 +91,9 @@ function compacttype(T::Type, maxwidth::Int=8, initial::Bool=true)
 
     maxwidth -= 1 # we will add "…" at the end
 
-    if T <: CategoricalValue
+    # This is only type display shortening so we
+    # are OK with any T whose name starts with Categorical here
+    if startswith(sT, "Categorical")
```

Comment:
We only need to handle `CategoricalValue`, right. so maybe better strictly check for `"CategoricalValue"`? Abbreviating other types could be weird.

Reply:
OK - I have written it when `CategoricalString` was present.